src/System.Data.Common/src/System/Data/Common/DbDataReader.cs

```diff
@@ -23,6 +23,11 @@ protected DbDataReader() : base()
         {
         }
 
+        ~DbDataReader()
```

Comment:
Yeah, I commented to that effect before I saw that. I'm going to split out both the PR and the issue accordingly.